docs/shared/guides/dynamic-mfe-angular.md

```diff
@@ -0,0 +1,167 @@
+  .then(() => import('./bootstrap').catch((err) => console.error(err)));
+```
+
+You’ll notice that we:
+Fetch the JSON file
+Call `setRemoteDefinitions` with the contents of the JSON file
+This tells webpack where each of our Remote applications has been deployed to!
+
+### Change how Remotes are loaded
+
+At the moment, webpack is statically building our application, telling it at build time where our Remotes are. That is because they are specified in the `mfe.config.js` file.
+
+Open the `mfe.config.js` file at the root of our `apps/dashboard/` folder and set the `remotes` property to be an empty array. It should look like this:
+
+```js
+module.exports = {
+  name: 'dashboard',
+  remotes: [],
+};
+```
+
+Next, we need to change how our application attempts to load the Remote when it is routed to. Open `app.module.ts` under the `src/app/`folder.
+
+You should see the following line in the `RouterModule.forRoot()`:
+
+```ts
+{
+    path: 'login',
+    loadChildren: () =>
+       import('login/Module').then((m) => m.RemoteEntryModule),
+}
+```
+
+Replace it with the following:
+
+```ts
+{
+    path: 'login',
+    loadChildren: () =>
+        loadRemoteModule('login', './Module').then(
+            (m) => m.RemoteEntryModule
+         ),
+}
+```
+
+You will also need to add the following import to the top of the file:
+
+```ts
+import { loadRemoteModule } from '@nrwl/angular/mfe';
+```
+
+The `loadRemoteModule` helper method simply hides some logic that will check if the Remote application has been loaded, and if not, load it, and then requests the correct exposed module from it.
+
+### Summary
+
+That’s all the changes required to replace Static Module Federation with Dynamic Module Federation.
+
+Running:
+
+```bash
+nx serve-mfe dashboard
+```
+
+Should result in the same behaviour as before, except that our Dashboard application is waiting until runtime to find out the deployed location of our Login application.
+
+In the next section, we will see how Nx’s generators can be used to automate a lot of this process for us!
+
+---
+
+## Creating a new Dynamic Host application
+
+Nx provides generators that aim to streamline the process of setting up a Dynamic Micro Frontend architecture.
+
+To showcase this, let’s create a new Host application that will use our previous Login application as well as a new Todo Remote application.
+
+### Generate Employee Host application
+
+Run the following command to generate a new Host application that is preconfigured for Dynamic Federation and add specify the Login Remote application we want to add:
+
+```bash
+nx g @nrwl/angular:host employee --remotes=login --dynamic
+```
+
+This will generate:
+Angular application
+Webpack Configuration _(webpack.config.js)_
+Module Federation Configuration _(mfe.config.js)_
+Micro Frontend Manifest File _(mfe.manifest.json)_
+Changes to the bootstrap of application to fetch the Micro Frontend Manifest, set the Remote Definitions and load the Login application correctly
```

Comment:
```suggestion
This will generate:

- An Angular application
- A Webpack configuration file _(webpack.config.js)_
- A Module Federation configuration file _(mfe.config.js)_
- A Micro Frontend Manifest file _(mfe.manifest.json)_
- Changes to the bootstrap of the application to fetch the Micro Frontend Manifest, set the Remote Definitions, and load the Login application correctly
```